tests/phpt/anon_classes/003_constructor_arguments.phpt

```diff
@@ -0,0 +1,21 @@
+@ok
+<?php
+
+function test_one_ctor_arg() {
+  new class(77) {
+    function __construct($num) {
```

Comment:
Please, add a @kphp_should_fail test like
```
  $int = 77;
  new class($int) {
    function __construct(string $num) { }
  }
```
(an error in types)
Let's see whether this error points to a correct location and stacktrace.

Reply:
Done, please check [comment](https://github.com/VKCOM/kphp/pull/273/files#r662103550).